src/containers/Output.js

```diff
@@ -0,0 +1,17 @@
+import {connect} from 'react-redux';
+import Output from '../components/Output';
+import {
+  isDraggingColumnDivider,
+  getOpenTopBarMenu,
+} from '../selectors';
+
+function mapStateToProps(state) {
+  return {
+    isDraggingColumnDivider: isDraggingColumnDivider(state),
+    openTopBarMenu: getOpenTopBarMenu(state),
```

Comment:
I would call this something like `isAnyTopBarMenuOpen` and cast it to a boolean? (I think as written this would fail the PropType check)